Assets/_Project/Scripts/Player/ILocalisationProvider.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections;
+using System.Collections.Generic;
+using Mapbox.Utils;
+using UnityEditor.UIElements;
+using UnityEngine;
+
+namespace PolSl.UrbanHealthPath.Map
+{
+    public interface ILocalizationProvider
```

Comment:
Based on ILocationProvider from Mapbox, I think that we'll also need to receive both Coordinates and angle the user is heading, so this should probably return custom "Location" instance.